WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -69,6 +69,7 @@ def perform
     GsuiteMailingLists.sync_group("reports@worldcubeassociation.org", User.delegate_reports_receivers_emails)
     GsuiteMailingLists.sync_group("advisory@worldcubeassociation.org", Team.wac.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("sports@worldcubeassociation.org", Team.wsot.current_members.includes(:user).map(&:user).map(&:email))
+    GsuiteMailingLists.sync_group("archive@worldcubeassociation.org ", Team.wat.current_members.includes(:user).map(&:user).map(&:email))
```

Comment:
Nitpick: There's a superfluous space at the end of the email address

Reply:
This typo also explains the current test failure

Reply:
got it thanks!